draft-ppm-protocol.md

```diff
@@ -728,11 +731,15 @@ struct {
 ~~~
 
 The `time`, `nonce`, and `extensions` fields have the same value as those in the
-report uploaded by the client. Similarly, the `helper_share` field is the helper's
-encrypted input share as it appeared in the report. [OPEN ISSUE: We usually only
-need to send this in the first aggregate request. Shall we exclude it in
-subsequent requests somehow?] The remainder of the structure is dedicated to the
-protocol-specific request parameters.
+report uploaded by the client. Similarly, the `helper_share` field is the
+`EncryptedInputShare` from the `Report` whose index in
+`Report.encrypted_input_shares` is equal to the index of `[aggregator]` in
+`Param.aggregator_endpoints`.
+
```

Comment:
Do you intend to start a new paragraph here? It's usually OK to inline OPEN ISSUEs with surrounding text.